src/client/uicommon.js

```diff
@@ -1,11 +1,12 @@
+import AudioManager from './audiomanager';
 import DRAW_IMAGE from './drawimage';
 import GameCanvas from './gamecanvas';
-import WZManager from './wzmanager';
 import PLAY_AUDIO from './playaudio';
+import WZManager from './wzmanager';
 
 const UICommon = {};
 
-UICommon.initialize = async function() {
+UICommon.initialize = async function () {
```

Comment:
Please no whitespace between `function` and the parentheses.

Reply:
@johncintron according to airbnb styleguide, the space is correct: 
space-before-function-paren